packages/kit/test/apps/basics/src/routes/errors/_tests.js

```diff
@@ -211,4 +211,28 @@ export default function (test, is_dev) {
 			assert.ok(!has_stack_trace, 'Stack trace is visible');
 		}
 	});
+
+	test(
+		'server-side 4xx status without error from load()',
+		'/errors/load-status-without-error-server',
+		async ({ page, response }) => {
+			assert.equal(await page.textContent('footer'), 'Custom layout');
+			assert.equal(
+				await page.textContent('#message'),
+				'This is your custom error page saying: "Internal Server Error"'
+			);
+			assert.equal(response.status(), 401);
+		}
+	);
+
+	test(
+		'client-side 4xx status without error from load()',
```

Comment:
I'm curious, is there a reason this test has to be different from https://github.com/sveltejs/kit/pull/1355/files#diff-ac526ac527f7cd84aec0ca97b27f1e8cbc7ad3f259e5833fd8c3055b52f60a5eR130 ?

Reply:
It doesn't have to be different. I needed to change it to body in troubleshooting to figure out what was going on because the test was failing. Turned out that the tests were failing because I needed to run `pnpm build` before testing. I thought the build was automatically being done before the test was run.

I added the original tests from https://github.com/sveltejs/kit/pull/1355/files#diff-ac526ac527f7cd84aec0ca97b27f1e8cbc7ad3f259e5833fd8c3055b52f60a5eR130  with the necessary changes